docker-compose.yml

```diff
@@ -1073,56 +1124,90 @@ services:
       - MARIADB_AUTO_UPGRADE=1
     command: "mysqld --innodb-flush-method=fsync"
 
-  mongodb:
-    image: mongo:8.0.10
-    container_name: appwrite-mongodb
+  # mongodb:
+  #   image: mongo:8.0.10
+  #   container_name: appwrite-mongodb
+  #   <<: *x-logging
+  #   networks:
+  #     - appwrite
+  #   volumes:
+  #     - appwrite-mongodb:/data/db
+  #     - appwrite-mongodb-keyfile:/data/keyfile
+  #     - ./mongo-init.js:/docker-entrypoint-initdb.d/mongo-init.js:ro
+  #     - ./mongo-entrypoint.sh:/mongo-entrypoint.sh:ro
+  #   ports:
+  #     - "27017:27017"
+  #   environment:
+  #     - MONGO_INITDB_ROOT_USERNAME=root
+  #     - MONGO_INITDB_ROOT_PASSWORD=${_APP_DB_ROOT_PASS}
+  #     - MONGO_INITDB_DATABASE=${_APP_DB_SCHEMA}
+  #     - MONGO_INITDB_USERNAME=${_APP_DB_USER}
+  #     - MONGO_INITDB_PASSWORD=${_APP_DB_PASS}
+  #   entrypoint: ["/bin/bash", "/mongo-entrypoint.sh"]
+  #   healthcheck:
+  #     test: |
+  #       bash -c "
+  #         if mongosh -u root -p ${_APP_DB_ROOT_PASS} --authenticationDatabase admin --quiet --eval 'rs.status().ok' 2>/dev/null | grep -q 1; then
+  #           exit 0
+  #         else
+  #           mongosh -u root -p ${_APP_DB_ROOT_PASS} --authenticationDatabase admin --quiet --eval \"
+  #             rs.initiate({_id: 'rs0', members: [{_id: 0, host: 'appwrite-mongodb:27017'}]})
+  #           \" 2>/dev/null || exit 1
+  #         fi
+  #       "
+  #     interval: 10s
+  #     timeout: 10s
+  #     retries: 10
+  #     start_period: 30s
+
+  # appwrite-mongo-express:
+  #   image: mongo-express
+  #   container_name: appwrite-mongo-express
+  #   networks:
+  #     - appwrite
+  #   ports:
+  #     - "8082:8081"
+  #   environment:
+  #     ME_CONFIG_MONGODB_URL: "mongodb://root:${_APP_DB_ROOT_PASS}@appwrite-mongodb:27017/?replicaSet=rs0&directConnection=true"
+  #     ME_CONFIG_BASICAUTH_USERNAME: ${_APP_DB_USER}
+  #     ME_CONFIG_BASICAUTH_PASSWORD: ${_APP_DB_PASS}
+  #   depends_on:
+  #     - mongodb
+
+
+  postgresql:
+    build:
+      context: .
+      dockerfile: postgres.dockerfile
```

Comment:
This is fine in dev, in prod we'll need to create a repo with a custom image like we do for MySQL